rpcs3/rpcs3qt/emu_settings.cpp

```diff
@@ -318,7 +318,7 @@ void emu_settings::EnhanceCheckBox(QCheckBox* checkbox, SettingsType type)
 	});
 }
 
-void emu_settings::EnhanceSlider(QSlider* slider, SettingsType type)
+void emu_settings::EnhanceSlider(QSlider* slider, SettingsType type, int multiplier)
```

Comment:
why use this weird multiplier. why not just use mb with a propper stepping.

Reply:
I disagree, but I guess you're the boss. I'll set it to range of 10k and tick marks every 1024 I guess.

Reply:
I just don't see why I should be limited to 10 values when I can use all 10240 values in MB.
Also the max size of a cache isnt exactly something you need to pick exactly